webpack.common.js

```diff
@@ -1,29 +1,71 @@
 const path = require('path');
+const webpack = require('webpack');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const assets = './assets/src/';
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const combineLoaders = require('webpack-combine-loaders');
+const autoprefixer = require('autoprefixer');
 const externals = {
     jquery: "jQuery",
-    eejs: "eejs"
+    '@eventespresso/eejs' : 'eejs'
 };
+const reactVendorPackages = [
+    'react',
+    'react-dom',
+    'react-redux',
+    'redux',
+    'classnames',
+    'lodash'
+];
 /** see below for multiple configurations.
 /** https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations */
 config = [
+    {
+        externals: {
+          '@eventespresso/eejs': {
+            this: 'eejs',
+          }
+        },
+        entry: {
+          eejs : [
+            assets + 'eejs/index.js'
+          ]
+        },
+        module: {
+          rules: [
+            { test: /\.js$/, exclude: /node_modules/, loader: "babel-loader" }
+          ]
+        },
+        output: {
+          filename: '[name].[chunkhash].dist.js',
+          path: path.resolve(__dirname, 'assets/dist'),
+          library: ['eejs'],
+          libraryTarget: 'this'
+        },
+    },
     {
         entry: {
+            reactVendor: reactVendorPackages,
             //example
             // 'ee-shortcode-blocks': [
             //     assets + 'blocks/index.js'
             // ],
         },
         plugins: [
             new CleanWebpackPlugin(['assets/dist']),
-            new ExtractTextPlugin('ee-[name].dist.css')
+            new ExtractTextPlugin('ee-[name].[contenthash].dist.css'),
```

Comment:
This requires the use of `[contenthash]` instead of `[chunkhash]` because if I use `[chunkhash]` that will just inherit whatever was generated for the base js containing the css that is extracted.  `[contenthash]` is a command-reference specific to the `ExtractTextPlugin`.